app/controllers/api/v1/twilio_controller.rb

```diff
@@ -3,13 +3,47 @@
 
 module Api::V1
   class TwilioController < Api::V1::ApiController
-    include Webhookable
+    include TwilioConfig
 
-    after_filter :set_header, except: [:assignment, :hold_music]
+    after_filter :set_header, except: [:assignment, :hold_music, :twilio_generate_call_token]
     skip_authorization_check
     skip_before_action :validate_token
     skip_before_action :verify_authenticity_token
 
+    # OUTBOUND CALL : START
+    def connect_outbound_call
+      caller_id, offer_id, mobile = params["phone_number"].split("#")
```

Comment:
We are passing in ```caller_id```, ```offer_id``` and ```mobile``` here but do we really need all this information? Would it be better simply to send ```offer_id``` and work out the ```caller_id``` as the authenticated donor and ```mobile``` from ```offer.created_by.mobile```

This would mitigate any param injection attacks and avoid duplication.